### PR TITLE
Security enhancements for data exhaust APIs - configuration changes

### PR DESCRIPTION
**Pull request by sowmya-dixit**

### CODE REVIEW

File: ansible/roles/stack-sunbird/defaults/main.yml [hunk 1]
```diff
@@ -752,3 +752,5 @@ job_request: "{{ env_name }}_job_request"
 
 #### Exhuats API consumer 
 exhaust_api_consumer_ids: ["273f3b18-5dda-4a27-984a-060c7cd398d3"] # being used in analytics api to access the analytics exhaust api
+user_profile_read_url: "https://{{ domain_name }}/api/user/v2/read/" # being used in analytics api to get user details
+org_search_url: "https://{{ domain_name }}/api/org/v1/search" # being used in analytics api to get mhrd tenant id
```
Comment (SanthoshVasabhaktula) [inline ansible/roles/stack-sunbird/defaults/main.yml:756]: call should be done via internal service

**Review by SanthoshVasabhaktula — CHANGES_REQUESTED** [review]

File: ansible/roles/stack-sunbird/defaults/main.yml (hunk 1 above)
Comment (sowmya-dixit) [reply ansible/roles/stack-sunbird/defaults/main.yml:756]: For this change will give new PR

File: ansible/roles/stack-sunbird/templates/analytics_api_service.conf [hunk 2]
```diff
@@ -156,6 +156,10 @@ elasticsearch.index.compositesearch.name="compositesearch"
 elasticsearch.index.dialcodemetrics.name="dialcodemetrics"
 metrics.dialcodemetrics.request.limit=1000
 dataexhaust.authorization_check=true
+user.profile.url="{{ user_profile_read_url }}"
+org.search.url="{{ org_search_url }}"
+standard.dataexhaust.roles=["ADMIN","ORG_ADMIN","REPORT_ADMIN"]
```
Comment (SanthoshVasabhaktula) [inline ansible/roles/stack-sunbird/templates/analytics_api_service.conf:161]: Roles should be ORG_ADMIN,REPORT_ADMIN

File: ansible/roles/stack-sunbird/templates/analytics_api_service.conf [hunk 3]
```diff
@@ -156,6 +156,10 @@ elasticsearch.index.compositesearch.name="compositesearch"
...
+ondemand.dataexhaust.roles=["ADMIN","ORG_ADMIN","REPORT_ADMIN","COURSE_ADMIN"]
```
Comment (SanthoshVasabhaktula) [inline ansible/roles/stack-sunbird/templates/analytics_api_service.conf:162]: Here the roles should be ORG_ADMIN, REPORT_ADMIN, CONTENT_CREATOR, COURSE_MENTOR

**Review by SanthoshVasabhaktula — CHANGES_REQUESTED** [review]

File: ansible/roles/stack-sunbird/templates/analytics_api_service.conf (hunk 2 above)
Comment (sowmya-dixit) [reply ansible/roles/stack-sunbird/templates/analytics_api_service.conf]: Done - updated

File: ansible/roles/stack-sunbird/templates/analytics_api_service.conf (hunk 3 above)
Comment (sowmya-dixit) [reply ansible/roles/stack-sunbird/templates/analytics_api_service.conf]: Done - updated

**Review by SanthoshVasabhaktula — APPROVED** [review]

**Pull request merged**